docs/external-disk.md

```diff
@@ -19,45 +19,108 @@ The `-disk` specification may be repeated for multiple disks, although a limited
 
 **TODO:** GCP
 
-## Mount the Disk
-A disk created or used via `hyperkit run` will be available inside the image at `/dev/vda` with the first partition at `/dev/vda1`.
+## Format the disk
 
-In order to use the disk, you need to do several steps to make it available:
+`pkg/format` creates a partition table and format drives for use with LinuxKit
 
-1. Create a partition table if it does not have one.
-2. Create a filesystem if it does not have one.
-3. `fsck` the filesystem.
-4. Mount it.
+### Example Usage
 
-To simplify the process, two `onboot` images are available for you to use:
+This packages supports two modes of use:
 
-1. `format`, which:
-    * checks for a partition table and creates one if necessary
-    * checks for a filesystem on the partition and creates one if necessary
-    * runs `fsck` on the filesystem
-2. `mount` which mounts the filesystem to a provided path
+```
+onboot:
+  name: format
+  image: linuxkit/format:${hash}
+```
 
-```yml
+In this mode of operation, the first disk found that does not have a valid partition table 
+will have one linux partition created that fills the entire disk
+
+### Options
+
+```
 onboot:
-  - name: format
-    image: linuxkit/format:84a997e69051a1bf05b7c1926ab785bb07932954
-  - name: mount
-    image: linuxkit/mount:ac8939c4102f97c084d9ddfd445c1908fce6d768
-    command: ["/mount.sh", "/var/external"]
+  name: format
+  image: linuxkit/format:${hash}
+  command: ["/usr/bin/format", "-type", "ext4", "-label", "DATA", "/dev/vda"]
 ```
 
-Notice several key points:
+`-type` can be used to specify the type e.g ext4
+`-label` can be used to give the disk a label
+The final (optional) argument specifies the device name
```

Comment:
Does this also fill the entire disk like above? I assume that any more complex partition tables are not doable here, should either be done outside or once in LinuxKit running? Probably should add to the docs.